examples/crhmc_sampling/plot_crhmc.py

```diff
@@ -0,0 +1,116 @@
+# VolEsti (volume computation and sampling library)
```

Comment:
is this file different from plot_hmc.py? if no, since this is used in quite a few places in the software, I suggest moving it to the examples/ directory (or examples/scripts). @vissarion what do you think?

Reply:
I added also a heatmap representation for two dimensions.

Reply:
I moved the scripts from logconcave and crhmc to `examples/python_utilities` while updating the ReadMe files